src/needs-parens.js

```diff
@@ -162,10 +162,7 @@ function needsParens(path) {
         (parent.init.includes(node) || parent.increment.includes(node))
       ) {
         return false;
-      } else if (
-        ["if", "while", "do", "switch"].includes(parent.kind) &&
-        name === "test"
-      ) {
+      } else if (parent.kind === "while" && node.left.kind === "list") {
```

Comment:
We need allow print `while` without parens in this case:
```
while (list($id, $name, $salary) = $result->fetch(PDO::FETCH_NUM)) {
    echo " <tr>\n" .
          "  <td><a href=\"info.php?id=$id\">$name</a></td>\n" .
          "  <td>$salary</td>\n" .
          " </tr>\n";
}
```

From http://php.net/manual/en/function.list.php.
Many people use this construction.